mkdocs.yml

```
@@ -60,6 +60,7 @@ nav:
         - Comparing PoA Consensus Protocols: Consensus-Protocols/Comparing-PoA.md
         - Clique: Consensus-Protocols/Clique.md
         - IBFT 2.0: Consensus-Protocols/IBFT.md
+        - IBFT 1.0: Consensus-Protocols/QuorumIBFT.md
```

Perhaps showing Quorum in the page title for nav.
```suggestion
        - Quorum IBFT 1.0: Consensus-Protocols/QuorumIBFT.md
```

---

Fixed